projects/plugins/jetpack/changelog/fix-my-sites-on-atomic-without-sso

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: bugfix
+
+Fix My Sites navigation in WP-Admin for Atomic sites with SSO disabled (and nav-unification disabled).
```

Comment:
Nitpick: These changelogs are public on each Jetpack release, and _Atomic_ is used only internally. We could maybe omit the changelog for this changeset, given that it only affects Atomic sites.